tests/test_integration.py

```diff
@@ -88,15 +88,17 @@ def test_update_schemas(self):
         #models.CaniddateCommitteeTotalsHouseSenate passed integration, so testing this specific
         #model really isn't expanding code coverage.  I can try and get the model passing but it's proving
         #to be difficult considering the joins needed (and our limited test subset)
-        whitelist = [models.EntityReceiptDisbursementTotals]
+        whitelist = [
+            models.EntityReceiptDisbursementTotals,
+            models.itemized.ScheduleA,
+            models.itemized.ScheduleB,
+        ]
```

Comment:
As discussed, totally appropriate.  These models are not managed by the `update_schema` process.